app.js

```diff
@@ -91,10 +91,7 @@ const UserSchema = new mongoose.Schema({
 		unique: true,
 		required: true
 	},
-	password: {
-		type: String,
-		required: true
-	}
+	password: String
```

Comment:
I'd like to understand why the password is a simple string instead of an object
(as a separate note, in the future it would be best to store only encrypted passwords in the DB using a library like bcrypt, but let's make sure everything is working first)

Reply:
Passport automatically hashes it. That is why it is passed as a second argument. You can see in the DB that salt and hash are log strings that dont actually match. I have heard of bcrypt and head it is good though.